main.py

```diff
@@ -71,7 +71,20 @@ class HomePageRedirectHandler(base.BaseHandler):
     """
     def get(self):
         if self.user_id and user_services.has_fully_registered(self.user_id):
-            self.redirect(feconf.DASHBOARD_URL)
+
+            # Get all the explorations associated with the user's account
+            user_exps = user_services.get_user_contributions(self.user_id)
+
+            # If the user is not learner it will have atleast
+            # one exloration associated with the user account
+            if len(user_exps.created_exploration_ids) >= 1:
+
```

Comment:
You don't need all the extra blank lines here, and some of the comments are unnecessary (see above).